src/components/va-accordion/va-accordion.tsx

```diff
@@ -25,15 +25,22 @@ import { Component, Element, Host, Listen, Prop, h } from '@stencil/core';
 export class VaAccordion {
   @Element() el!: any;
 
+  @Event({
+    eventName: 'component-library-analytics',
+    composed: true,
+    bubbles: true,
+  })
+  componentLibraryAnalytics: EventEmitter;
+
   @Listen('accordionItemToggled')
   itemToggledHandler(event: CustomEvent) {
     // The event target is the button, and it has a parent which is a header.
     // It is the parent of the header (the root item) that we need to access
     // The final parentNode will be a shadowRoot, and from there we get the host.
     // https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot/host
     const clickedItem =
-      event.detail.target.parentNode.parentNode.host || // if we are on IE, `.host` won't be there
-      event.detail.target.parentNode.parentNode;
+      event.detail.currentTarget.parentNode.parentNode.host || // if we are on IE, `.host` won't be there
+      event.detail.currentTarget.parentNode.parentNode;
```

Comment:
FYSA @bkjohnson, I noticed this was a bit wonky if you clicked the subheader or whitespace.  `currentTarget` seems to do the trick.

Reply:
Interesting, thanks for the fix!